cinn/frontend/syntax.h

```diff
@@ -184,6 +184,7 @@ struct Program {
       : instrs_(std::move(instrs)), inputs_(std::move(inputs)) {}
 
   void SetInputs(const std::vector<Variable>& xs);
+  std::vector<Variable> GetInputs() { return inputs_; }
```

Comment:
返回vector的const应用是不是好一些？
```suggestion
  const std::vector<Variable>& GetInputs() const { return inputs_; }
```

Reply:
done